.github/e2e/docker-compose.yml

```diff
@@ -15,7 +16,7 @@ services:
       # 12D3KooWBM3SdXWqGaawQDGQ6JprtwswEg3FWGvGhmgmMez1vRbR
       - -k=hK62afickoeP2uZbmSkAYXxxqP8ozq16VRN7qfTP719EHC5V5tjrtW57BSjUr8GvsEXmJRbtejUWyPZ2rZMyQdq
     volumes:
-      - ../../aqua-ipfs:/.fluence/v1/builtins/aqua-ipfs
+      - ../../aqua-ipfs:/builtins/aqua-ipfs
```

Comment:
Since we are using aqua-ipfs as a cargo crate there is no need to mount a volume. Volumes should be deleted